internal/serverinstall/nomad.go

```diff
@@ -133,6 +141,54 @@ func (i *NomadInstaller) Install(
 		}, nil
 	}
 
+	if strings.ToLower(i.config.volumeType) == "csi" {
+		var (
+			// bytes
+			defaultCSIVolumeCapacityMin = int64(1073741824)
+			defaultCSIVolumeCapacityMax = int64(2147483648)
+		)
```

Comment:
These should probably be `const` at the file level, since it looks like they're establishing our expected defaults, and we generally put our expected defaults as consts at the top of the file.

Reply:
oooh, yeah I didn't make them global because I thought they weren't used outside this method, but this just reminded me I forgot to put them as defaults in the flags